build-and-test.ps1

```diff
@@ -22,12 +23,17 @@ $manifestRepo = $manifest.Repos[0]
 $activeOS = docker version -f "{{ .Server.Os }}"
 $builtTags = @()
 
+$buildFilter = $Filter
+if ($activeOS -eq "windows" -and $buildFilter -like "2.*") {
```

Comment:
Case-insensitive version of `-eq`. In case Docker return `Windows` instead of the current `windows`.

---
In reply to: [145519453](https://github.com/dotnet/dotnet-docker-nightly/pull/467#discussion_r145519453) [](ancestors = 145519453)

Reply:
`eq` is by default case-insensitive.  It is my understanding that `ieq` is only necessary if you want to explicitly call out the inequality.  IMO `eq` is sufficient here.

see [https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-5.1](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-5.1)
> By default, all comparison operators are case-insensitive. To make a comparison operator case-sensitive, precede the operator name with a c. For example, the case-sensitive version of -eq is -ceq. To make the case-insensitivity explicit, precede the operator with an i. For example, the explicitly case-insensitive version of -eq is -ieq.

---
In reply to: [145525177](https://github.com/dotnet/dotnet-docker-nightly/pull/467#discussion_r145525177) [](ancestors = 145525177,145519453)

Reply:
Ok. Let's leave it as-is.

---
In reply to: [145526838](https://github.com/dotnet/dotnet-docker-nightly/pull/467#discussion_r145526838) [](ancestors = 145526838,145525177,145519453)